catalog/resource_catalog_workspace_binding.go

```diff
@@ -0,0 +1,66 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type CatalogWorkspaceBindingInfo struct {
+	Name      string `json:"name" tf:"force_new"`
+	Workspace int64  `json:"workspace" tf:"force_new"`
+}
+
+func ResourceCatalogWorkspaceBinding() *schema.Resource {
```

Comment:
please use bind resource with the pair id. see example at https://github.com/databricks/terraform-provider-databricks/blob/master/scim/resource_group_role.go#L14-L30